src/plugins/modal.js

```diff
@@ -26,7 +26,8 @@ class Modal {
          */
         this.closeButton = this.findCloseButton();
 
-        this.setupCloseEvent();
+        if(this.closeButton || (options.hasOwnProperty('closable') && options.closable))
+            this.setupCloseEvent();
```

Comment:
Would you be able to update this so `closable` is it's own property on the `modal` class and so that the `if` statement has the curly brackets. I know it's not required but it's a style I'm trying to follow.

Reply:
Got it!